src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -46,8 +48,8 @@ function informationDiagram(publicAPI, model) {
 
   model.clientRect = null;
 
-  // FIXME: Make some attempt at unique id, for now just use millis timestamp
-  model.instanceID = `informationDiagram-${Date.now()}`;
+  // unique id, based on millis timestamp
+  model.instanceID = `informationDiagram-${generateUUID()}`;
```

Comment:
OK, seems fine. It's used for an xlink:href target ID, so it has to be unique in the entire page, but a count with the module string should be fine.